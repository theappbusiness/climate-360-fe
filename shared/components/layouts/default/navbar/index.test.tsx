import { render } from '@testing-library/react'
import { Navbar } from '.'

describe('Navbar', () => {
  it('matches the snapshot', () => {
    const { asFragment } = render(<Navbar />)
    expect(asFragment()).toMatchSnapshot()
  })
})
