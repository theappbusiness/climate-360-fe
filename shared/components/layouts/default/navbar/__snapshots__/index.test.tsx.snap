// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar matches the snapshot 1`] = `
<DocumentFragment>
  <nav
    class="css-n6fcxb"
  >
    <p
      class="chakra-text css-1h8lr2g"
    >
      Climate 360
    </p>
    <button
      class="chakra-button css-1efidcl"
      type="button"
    >
      üçî
    </button>
    <div
      class="css-8olkqp"
    >
      <div
        class="chakra-stack css-enti4i"
      >
        <a
          class="chakra-link css-0"
          href="/learn"
        >
          Learn
        </a>
        <a
          class="chakra-link css-0"
          href="/calculate-co2"
        >
          Carbon footprint
        </a>
        <a
          class="chakra-link css-0"
          href="/help"
        >
          Get help
        </a>
      </div>
    </div>
  </nav>
</DocumentFragment>
`;
